cmd/osbuild-composer/config_test.go

```diff
@@ -90,6 +98,13 @@ func TestConfig(t *testing.T) {
 	require.Equal(t, []string{"https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs"}, config.Koji.JWTKeysURLs)
 	require.Equal(t, "", config.Koji.JWTKeysCA)
 	require.Equal(t, "/var/lib/osbuild-composer/acl", config.Koji.JWTACLFile)
+
+	expectedDistroAliases := map[string]string{
+		"rhel-7": "rhel-7.9", // this value is from the default config
+		"rhel-8": "rhel-8.9",
```

Comment:
These don't match what's above. And I wonder if it would be possible to set the alias to the latest of each X release automatically? Or is there ever a reason for the alias to lag behind?

Reply:
One note to the question of if we can set the tag to the latest release automatically:
Previously, this was hard coded in the distro definition. There is no longer any notion of what is the latest release in the distro / image definitions. The idea is that distro versions with defined repositories are treated as tested and supported. We could probably check the available repositories for the on-prem and set the aliases to the latest release, but not for the service. In Cloud API, the list of repositories is provided with the compose request. Moreover, as mentioned above, there is a need to have different target defined for these aliases on-prem and in the service deployment.